pkg/generator/gen_deploy.go

```diff
@@ -22,40 +22,55 @@ import (
 )
 
 const (
+	crdTmplName      = "deploy/crd.yaml"
 	operatorTmplName = "deploy/operator.yaml"
 	rbacTmplName     = "deploy/rbac.yaml"
 	crTmplName       = "deploy/cr.yaml"
 )
 
-// OperatorYaml contains all the customized data needed to generate deploy/operator.yaml for a new operator
-// when pairing with operatorYamlTmpl template.
-type OperatorYaml struct {
+// CRDYaml contains data needed to generate deploy/crd.yaml
+type CRDYaml struct {
 	Kind         string
 	KindSingular string
 	KindPlural   string
 	GroupName    string
 	Version      string
-	ProjectName  string
-	Image        string
+}
+
+// renderCRDYaml generates deploy/crd.yaml
+func renderCRDYaml(w io.Writer, kind, apiVersion string) error {
+	t := template.New(crTmplName)
```

Comment:
I know the template name doesn't actually matter but this should be `crdTmplName`.

Reply:
@hasbro17 yeah, the name doens't matter. just for formality i guess. good catch on this one.